deps/CMakeLists.txt

```diff
@@ -74,9 +68,14 @@ if( POLICY CMP0037 )
   cmake_policy( SET CMP0037 OLD )
 endif( )
 
-add_custom_target( install
-  ${boost_custom_target}
+add_custom_target( build_deps
   ${gtest_custom_target}
   ${lapack_custom_target}
   DEPENDS ${hipsolver_dependencies}
 )
+
+add_custom_target( install_deps
+  ${gtest_install_target}
+  ${lapack_install_target}
+  DEPENDS ${build_deps}
```

Comment:
I didn't notice when I reviewed the rocblas code, but this doesn't make sense. `build_deps` is not a variable. It's a target name. I think `${build_deps}` evaluates to the empty string.